lib/active_model/serializer/associations.rb

```diff
@@ -80,10 +78,11 @@ def associate(reflection)
         end
       end
 
-      # @param [IncludeTree] include_tree (defaults to all associations when not provided)
+      # @param [IncludeTree] include_tree (defaults to the
+      #   default_includes config value when not provided)
       # @return [Enumerator<Association>]
       #
-      def associations(include_tree = DEFAULT_INCLUDE_TREE)
+      def associations(include_tree = ActiveModel::Serializer::IncludeTree.from_include_args(ActiveModel::Serializer.config.default_includes))
```

Comment:
would be better if the default were memoized so it doesn't need to be recalculated all the time.  I can't imagine changing the default after initialization.

```ruby
def self.default_include_tree
  @default_include_tree ||= ActiveModel::Serializer::IncludeTree.from_include_args(ActiveModel::Serializer.config.default_includes)
end

def associations(include_tree = self.class.default_include_tree)
```

would be a little better, assuming we're concerned about the file being evaluated before the `default_includes` is set in an initializer.

Also, I missed this, but it should be `s/ActiveModel::Serializer.config/ActiveModelSerializers.config` as we're moving away from the `Serializer` class being the primary namespace.  For now, it's just a reference, so, not wrong, just not the pattern we're moving towards.